documentation/IDTA-01002-3/modules/ROOT/pages/IDTA-01002_SpecificationAssetAdministrationShell_Part2_API.adoc

```diff
@@ -1,6643 +1,5602 @@
-    },
-    "value": [ {
-        "modelType": "Property",
-        "idShort": "MaxRotationSpeed",
-        "category": "PARAMETER",
-        "semanticId": {
-          "keys": [ {
-              "type": "ConceptDescription",
-              "value": "0173-1#02-BAA120#008"
-          } ],
-          "type": "ExternalReference"
-        },
-        "valueType": "xs:int",
-        "value": "5000"
-      } ]
-  } ]
-}
-
-----
-
-a|
-If applied to the Submodel:
-
-[source,json,linenums]
-----
-{
-  "modelType": "Submodel",
-  "id": "http://i40.customer.com/type/1/1/7A7104BDAB57E184",
-  "idShort": "TechnicalData",
-  "semanticId": {
-    "keys": [ {
-        "type": "GlobalReference",
-        "value": "0173-1#01-AFZ615#016"
-    } ],
-    "type": "ExternalReference"
-  },
-  "submodelElements": [ {
-    "modelType":  "SubmodelElementCollection",
-    "idShort": "RotationSpeed",
-    "semanticId": {
-      "keys": [ {
-          "type": "GlobalReference",
-          "value": "http://purl.org/iot/vocab/iot-taxonomy-lite#RotationalSpeed"
-      } ],
-      "type": "ExternalReference"
-    }
-  } ]
-}
-
-----
-
-|*Metadata* a|
-If applied to the Submodel:
-
-[source,json,linenums]
-----
-{
-  "modelType": "Submodel",
-  "id": "http://i40.customer.com/type/1/1/7A7104BDAB57E184",
-  "idShort": "TechnicalData",
-  "semanticId": {
-    "keys": [ {
-        "type": "GlobalReference",
-        "value": "0173-1#01-AFZ615#016"
-    } ],
-    "type": "ExternalReference"
-  },
-  "submodelElements": [ {
-    "modelType":  "SubmodelElementCollection",
-    "idShort": "RotationSpeed",
-    "semanticId": {
-      "keys": [ {
-          "type": "GlobalReference",
-          "value": "http://purl.org/iot/vocab/iot-taxonomy-lite#RotationalSpeed"
-      } ],
-      "type": "ExternalReference"
-    },
-    "value": [ {
-        "modelType": "Property",
-        "idShort": "MaxRotationSpeed",
-        "category": "PARAMETER",
-        "semanticId": {
-          "keys": [ {
-              "type": "ConceptDescription",
-              "value": "0173-1#02-BAA120#008"
-          } ],
-          "type": "ExternalReference"
-        },
-        "valueType": "xs:int"
-      } ]
-  } ]
-}
-
-----
-
-a|
-If applied to the Submodel:
-
-[source,json,linenums]
-----
-{
-  "modelType": "Submodel",
-  "id": "http://i40.customer.com/type/1/1/7A7104BDAB57E184",
-  "idShort": "TechnicalData",
-  "semanticId": {
-    "keys": [ {
-        "type": "GlobalReference",
-        "value": "0173-1#01-AFZ615#016"
-    } ],
-    "type": "ExternalReference"
-  },
-  "submodelElements": [ {
-    "modelType":  "SubmodelElementCollection",
-    "idShort": "RotationSpeed",
-    "semanticId": {
-      "keys": [ {
-          "type": "GlobalReference",
-          "value": "http://purl.org/iot/vocab/iot-taxonomy-lite#RotationalSpeed"
-      } ],
-      "type": "ExternalReference"
-    }
-  } ]
-}
-
-----
-
-
-If applied to the Property, i.e. idShortPath “RotationSpeed.MaxRotationS
```

Comment:
@BirgitBoss the old state had something like "annex B has a ValueOnly example", which I just changed to a more generic formulation.